weatherrouting/utils.py

```diff
@@ -99,4 +100,55 @@ def reduce180 (alfa):
 		return 0.0
 	return alfa
 
-	
+def pathAsGeojson(path):
+	tr = []
+	for wp in path:
+		if len(wp) == 3:
+			tr.append((wp[0], wp[1], str(wp[2]), 0, 0, 0, 0))
+		else:
+			tr.append((wp[0], wp[1], str(wp[4]), wp[5], wp[6], wp[7], wp[8]))
+
+	feats = []
+	route = []
+
+	for order,wayp in enumerate(tr):
+		feat = {
+			"type": "Feature",
+			"id": order,
+			"geometry": {
+				"type": "Point",
+				"coordinates": [
+					wayp[0],
+					wayp[1]
+				]
+			},
+			"properties": {
+				"timestamp": str(wayp[2]),
+				"attr1": wayp[3],
+				"attr2": wayp[4],
+				"attr3": wayp[5],
+				"attr4": wayp[6]
+			}
+		}
+		feats.append(feat)
+		route.append([wayp[0], wayp[1]])
+
+	feats.append({
+		"type": "Feature",
+		"id": order+1,
+		"geometry": {
+			"type": "LineString",
+			"coordinates": route
+		},
+		"properties": {
+			"start-timestamp": str(tr[0][2]),
+			"end-timestamp": str(tr[-1][2])
+		}
+	})
+
+	gj = {
+		"type": "FeatureCollection",
+		"features": feats
+	}
+
+	return json.dumps(gj)
```

Comment:
I think this should returns the gj object instead of the string, so this function is more generic and suitable for other usecases where a geojson object is required. Move the dumps in your print line.

Please comment out all print function, since they're only for debug, or use logging instead like this, so it can be turned on / off with one line of code:

```
log = logging.getLogger('test_2')

log.debug ('What you want to print for debugging')
```

Reply:
https://github.com/dakk/libweatherrouting/pull/2/commits/7bd70ea31b324bfec603372cb3d033120f769e92